File: Twitter.Domain/Entities/TwitterPost.cs
using NJsonSchema.Annotations;
using System;
using System.ComponentModel.DataAnnotations;

namespace Twitter.Domain.Entities
{
    public class TwitterPost
    {
        [Key]
        public Guid Id { get; set; }
        public string PostText { get; set; }
        [JsonSchemaDate]
        public DateTime DateCreation { get; set; }
        public int Like { get; set; }

        public string UserId { get; set; }
        public User User { get; set; }

        public Images Images { get; set; }
    }
}

File: TwitterAngular/src/app/models/twitter-post.ts
import { user } from "../models/user";
import { comment } from "./comment";
import { images } from "./images";

export class twitterPost{
    id:string;
    postText:string;
    dateCreation:Date;
    like:number;
    userId:string;
    user:user;
    images:images;
    comments:comment[];
}